packages/admin-ui/src/lib/login/src/components/login/login.component.ts

```diff
@@ -35,6 +46,26 @@ export class LoginComponent {
         });
     }
 
+    loadImage() {
+        this.httpClient
+            .get('https://login-image.vendure.io', {
+                params: new HttpParams().append('domain', window.location.hostname),
```

Comment:
Do we need to explicitly append the domain here? Is that not something that we'll get for free from the express request object? (If you are using Express for the image server).

Reply:
Well, the question is - is it always the same, when you are using the admin ui behind a CDN or load balancer?

Reply:
I think so, because this is a client-side app, the request will always come from the browser rather than the server hosting the admin ui app. So it _should_ be exactly the same as what you would get from `window.locations.hostname`, if my understanding is correct.